plugins/arangodbendorsements/main.go

```diff
@@ -892,15 +722,15 @@ func (e *ArangoEndorsementStore) GetMostRecentSwVerOfLinkedComp(args common.Quer
 	meas := measurements[0]
 	requiredswid, matched := checkMeasMatchInSwIDList(swidList, meas)
 	if !matched {
-		return nil, fmt.Errorf("query failed given platform identity has no matching measurements %v", err)
+		return nil, fmt.Errorf("query failed given platform identity has no matching measurements")
 	}
-	// Now we have the required swid, use it to find the best SW version to return
 
+	// Now we have the required swid, use it to find the best SW version to return
 	// Note by design it is assured that linked SW component (to platform) is always the base.
 	latestSwid := requiredswid
 	updateList, err := e.GetUpdatesListForBaseSwVersion(latestSwid)
 	if err != nil {
-		return nil, fmt.Errorf("unable to fecth queries from arangoDB %v", err)
+		return nil, fmt.Errorf("unable to fetch queries from store %v", err)
```

Comment:
for this message to be really useful, we should say which queries could not be fetched from the store.

Reply:
Added query in the error message as an argument!